### PR TITLE
added the option to disable google translations

### PR DESCRIPTION
**Pull request by following5**
I have set up a (devel) site and would like to see the original cache descriptions here (site lang is English, example descs are German).

### CODE REVIEW

File: viewcache.php [hunk 1]
```diff
@@ -1114,14 +1114,14 @@ function onTheList($theArray, $item)
...
-
         //is no description available in the wished language?
         if (array_search($desclang, $desclangs) === false) {
             $desclang = $desclangs[0];
         }
 
-        if (strtolower($desclang) != $lang && $lang != 'pl')
+        if (isset($disable_google_translation) && $disable_google_translation)
+            $enable_google_translation = false;
+        else if (strtolower($desclang) != $lang && $lang != 'pl')
```
Comment (wloczynutka) [inline viewcache.php]: Could you check why 'pl' is hardcoded? i suppouse this is a bug. Maybe it is possible to fix it?

File: viewcache.php [hunk 2]
```diff
@@ -1114,14 +1114,14 @@ function onTheList($theArray, $item)
...
 
-        $enable_google_translation = false;
-
         //is no description available in the wished language?
         if (array_search($desclang, $desclangs) === false) {
             $desclang = $desclangs[0];
         }
 
-        if (strtolower($desclang) != $lang && $lang != 'pl')
+        if (isset($disable_google_translation) && $disable_google_translation)
```
Comment (wloczynutka) [inline viewcache.php]: if / else should be always used witch braces { } (PSR-2 http://www.php-fig.org/psr/psr-2/)

File: viewcache.php (hunk 1 above)
Comment (following5) [inline viewcache.php]: I have no idea why _$lang != pl_ was added. Effectively, it disables automatic translation if the user choses Polish language. Does not look like a bug but some poor coding.

File: viewcache.php (hunk 2 above)
Comment (following5) [inline viewcache.php]: The original code was without braces. I have adjusted it to PSR-2 now.

**Comment by wloczynutka** [comment]
Hello everyone. I'm merging this PR, but please could you check if automatic google translation is really working anywhere? I remember that Google tranmsaltion was not working some time ago, but later I don't use it so I don't know.

**Comment by kojoty** [comment]
translations are not working - for reference the issue is here  #214

**Comment by andrixnet** [comment]
I think I also discussed at some point the fact that automated translations with Google Translate do not work.

**Pull request merged**